### ISSUE TITLE
Raise error instead of returning 0.0.0?

### ISSUE THREAD

**Issue by grst** [flying-sheep/get_version#7]
Related to #6 I was wondering if it was better to raise an Error instead of returning `0.0.0` if no version is found...

**Comment by flying-sheep** [comment]
Right. I don’t know why I did it that way. Maybe because versioneer did it that way?

**Comment by flying-sheep** [comment]
THis is long done!
